java/src/Ice/src/main/java/com/zeroc/Ice/UnknownLocalException.java

```diff
@@ -22,23 +9,11 @@
  * exceptions derived from {@link RequestFailedException}, which are transmitted by the Ice protocol
  * even though they are declared <code>local</code>.
  */
-public class UnknownLocalException extends UnknownException {
-  public UnknownLocalException() {
-    super();
-  }
-
-  public UnknownLocalException(Throwable cause) {
-    super(cause);
-  }
-
+public final class UnknownLocalException extends UnknownException {
   public UnknownLocalException(String unknown) {
```

Comment:
Should be message, not unknown.

Reply:
Fixed, we only have a single constructor, which only takes a `String message`.